cpp/src/ray/util/process_helper.cc

```diff
@@ -45,7 +55,9 @@ static void StopRayNode() {
 void ProcessHelper::RayStart(std::shared_ptr<RayConfig> config,
                              CoreWorkerOptions::TaskExecutionCallback callback) {
   std::string redis_ip = config->redis_ip;
-  if (config->worker_type == WorkerType::DRIVER && redis_ip.empty()) {
+  if (!redis_ip.empty()) {
+    ConnectRayNode(redis_ip, config->redis_password);
```

Comment:
We already support this in StartRayNode. And in fact, we already support connect a existing cluster which has no test in bazel.